File: O/source_code_2.cpp
#include <iostream>
	using namespace std;

	int main() {
	    int A[] = {1, 2, 3};
	    int B[] = {1, 2};
 	   int matrix[3][2] = {0};

  	  cout << "Ordered pairs in relation R (a > b):\n";
   	 for (int i = 0; i < 3; i++) {
    	    for (int j = 0; j < 2; j++) {
    	        if (A[i] > B[j]) {
    	            cout << "(" << A[i] << ", " << B[j] << ")\n";
     	           matrix[i][j] = 1;
      	      }
      	  }
   	 }

   	 cout << "\nMatrix Representation:\n";
   	 for (int i = 0; i < 3; i++) {
   	     for (int j = 0; j < 2; j++) {
   	         cout << matrix[i][j] << " ";
    	    }
     	   cout << endl;
   	 }

   	 return 0;
	}

File: O/source_code_2.py
A = [1, 2, 3]
B = [1, 2]

R = [(a, b) for a in A for b in B if a > b]
matrix = [[1 if (a, b) in R else 0 for b in B] for a in A]

print("Relation R (a > b):", R)
print("Matrix Representation:")
for row in matrix:
    print(row)
